### ISSUE TITLE
NTR: 2,4-dinitrophenylhydrazine urine test positive (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8794]
New term request 2,4-dinitrophenylhydrazine urine test positive (f348): DNPH  UMLS: 1855377

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000124
- label: Positive 2,4-dinitrophenylhydrazine urine test
- parents: [Elevated urine keto acid level (HP:0033456)](https://hpo.jax.org/app/browse/term/HP:0033456)
- pmid: PMID:32491705
- def: Positive (abnormal) result of the 2,4-dinitrophenylhydrazine urine test, which indicates the presence of alpha-ketoacids in the urine.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8794
- synonyms:
